### ISSUE TITLE
None of the failed examples for 'Element with role attribute has required states and properties' are failures

### ISSUE THREAD

**Issue by dd8** [act-rules/act-rules.github.io#1173]
`aria-controls` should only be set for `role=combobox` when the combobox popup is visible and ` aria-expanded=true`:
https://www.w3.org/TR/wai-aria-practices-1.1/#wai-aria-roles-states-and-properties-6

So these aren't a failures because the default state is ` aria-expanded=false`:
https://act-rules.github.io/rules/4e8ab6#failed-example-1
https://act-rules.github.io/rules/4e8ab6#failed-example-2

**Comment by Jym77** [comment]
I guess an easy fix would be to change these to `<div role="heading">` without an `aria-level`.

That would keep the examples simple.

**Comment by Jym77** [comment]
Closed by #1281.

**Comment by WilcoFiers** [comment]
Issue should not be closed until we the PR has been merged.
